momento-sdk/src/main/java/momento/sdk/ScsDataGrpcStubsManager.java

```diff
@@ -25,22 +25,65 @@ final class ScsDataGrpcStubsManager implements Closeable {
   private final ManagedChannel channel;
   private final ScsGrpc.ScsFutureStub futureStub;
   private final Duration deadline;
+  private final ScheduledExecutorService retryScheduler;
+  private final ExecutorService retryExecutor;
+
+  // An arbitary selection of twice the number of available processors.
+  private static final int MAX_RETRY_THREAD_POOL_SIZE =
```

Comment:
I'm not sure it makes sense to bound this based on the number of processors? That would definitely make sense for a cpu-bound thread pool but this is an IO-bound thread pool.

In Kotlin they default it to 64: https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-dispatchers/-i-o.html

Reply:
Good point. No opinion on the number in general so use 64